### PR TITLE
CHANGELOG: release 0.107.0

### PR DESCRIPTION
**Pull request by AnnaShaleva**
N3/NeoFS mainnet/testnet states match, privnet is OK.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,96 @@
 
 This document outlines major changes between releases.
 
+## 0.107.0 "Multithreading/Multiplication" (02 Dec 2024)
```
Comment (AnnaShaleva) [inline CHANGELOG.md:5]: Voting is open, suggestions are welcome. Multiplication is boring, Multithreading doesn't fit the naming policy.

Comment (roman-khimov) [reply CHANGELOG.md:5]: mongrelization

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,96 @@
...
+
+A large update that except for numerous enhancements and bug fixes introduces a
```
Comment (roman-khimov) [inline CHANGELOG.md:7]: `that except` doesn't read well

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,96 @@
...
+major node extension: NeoFS BlockFetcher service and `util upload-bin` CLI command
+implemented as a part of [NeoFS snapshot storage
+proposal](https://github.com/neo-project/neo/issues/3463). BlockFetcher service, as
+an alternative to P2P synchronisation mechanism, allows to download blocks and sync
+chain from block dumps stored in NeoFS. Starting from this release, NeoSPCC team
+maintains chain dumps in NeoFS for N3 and NeoFS public networks. Default NeoGo node
+configuration for these networks has been changed to use NeoFS BlockFetcher as a
```
Comment (roman-khimov) [inline CHANGELOG.md:14]: the default

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,96 @@
...
+default synchronisation start point prior to P2P synchronisation. Other than that,
```
Comment (roman-khimov) [inline CHANGELOG.md:15]: synchronization mechanism

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,96 @@
...
+
+New features:
+ * new NeoFS BlockFetcher service that allows to sync node from NeoFS chain dump
+   (#3515, #3636, #3637, #3632, #3691, #3706, #3668, #3713)
+ * new `util upload-bin` CLI command that allows to upload blocks from Neo chains to
+   NeoFS (#3578, #3625, #3626, #3633, #3637, #3638, #3643, #3650, #3662, #3684,
+   #3686, #3691)
+ * new `delete` and `ib` VM CLI commands for brealpoints management (#3674)
+ * NEP-24 standard support (#3560)
+ * Echidna hardfork introduced, but not yet enabled (#3554)
```
Comment (roman-khimov) [inline CHANGELOG.md:40]: Maybe signify that it's a Echidna preview, it's just a part of changes, it'll change in an incompatible way before the full support and can be enabled for experiments only.

File: CHANGELOG.md (hunk 2 above)
Comment (AnnaShaleva) [reply CHANGELOG.md]: Removed, we have enough details for other features of this release.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**